src/forms/StripePaymentForm/StripePaymentForm.js

```diff
@@ -167,8 +175,12 @@ class StripePaymentForm extends Component {
       errors,
       invalid,
       handleSubmit,
+      disabled,
+      form,
     } = formRenderProps;
 
+    this.form = form;
```

Comment:
Maybe `this.formAPI` or `this.finalFormAPI` instead?
https://github.com/final-form/react-final-form#form-formapi

Reply:
Also, initialize it with null in constructor.